components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -1359,7 +1367,7 @@ public static String createEndpoint(String endpointUrl, Registry registry) throw
                     String type=storeElem.getAttributeValue(new QName(APIConstants.EXTERNAL_API_STORE_TYPE));
                     store.setType(type); //Set Store type [eg:wso2]
                     String name=storeElem.getAttributeValue(new QName(APIConstants.EXTERNAL_API_STORE_ID));
-                    if(name==null){
+                    if (name == null) {
                         try {
                             throw new APIManagementException("The ExternalAPIStore name attribute is not defined in api-manager.xml.");
```

Comment:
Please fix this as well. Its not your change, but better to fix it. This throws and catches the same exception